integreat_cms/api/v3/social_media_headers.py

```diff
@@ -40,13 +40,19 @@
 
 def get_excerpt(content: str) -> str:
     """
-    Correctly escapes, truncates and normalizes the content of the page to display in a search result
+    Correctly escapes, truncates and normalizes the content of the page to display in a search result.
+    Then apply some crazy string operations to handle several edge cases.
 
     :param content: The content of the page
 
     :return: A page excerpt containing the first 100 characters of "raw" content
     """
-    return unescape(strip_tags(content))[:100].replace("\n", " ").replace("\r", "")
+    stripped_content = unescape(
+        strip_tags(content.replace("\n", " ").replace("\r", "").replace("<br>", " "))
+    )
+    if len(stripped_content) <= 100:
+        return stripped_content.strip().replace("  ", " ")
```

Comment:
Why do we need this line? :)

Reply:
For example 
```
Hallo<br>
Welt
```
will evaluate to `Hallo  Welt` as we replaced newlines and br tags with empty spaces.